--- cal3d-0.11.0/src/cal3d/hardwaremodel.cpp~	2004-12-01 23:42:18.000000000 +0100
+++ cal3d-0.11.0/src/cal3d/hardwaremodel.cpp	2008-11-25 10:39:33.000000000 +0100
@@ -8,6 +8,8 @@
 // your option) any later version.                                            //
 //****************************************************************************//
 
+#include <string.h>
+
 #ifdef HAVE_CONFIG_H
 #include "config.h"
 #endif
--- cal3d-0.11.0/src/cal3d/platform.cpp~	2003-12-07 19:22:39.000000000 +0100
+++ cal3d-0.11.0/src/cal3d/platform.cpp	2008-11-25 10:50:40.000000000 +0100
@@ -8,6 +8,8 @@
 // your option) any later version.                                            //
 //****************************************************************************//
 
+#include <string.h>
+
 #ifdef HAVE_CONFIG_H
 #include "config.h"
 #endif
--- cal3d-0.11.0/src/cal3d/renderer.cpp~	2004-12-30 20:20:54.000000000 +0100
+++ cal3d-0.11.0/src/cal3d/renderer.cpp	2008-11-25 10:51:06.000000000 +0100
@@ -8,6 +8,8 @@
 // your option) any later version.                                            //
 //****************************************************************************//
 
+#include <string.h>
+
 #ifdef HAVE_CONFIG_H
 #include "config.h"
 #endif
--- cal3d-0.11.0/src/cal3d/submesh.cpp~	2004-12-01 23:42:18.000000000 +0100
+++ cal3d-0.11.0/src/cal3d/submesh.cpp	2008-11-25 10:51:21.000000000 +0100
@@ -8,6 +8,8 @@
 // your option) any later version.                                            //
 //****************************************************************************//
 
+#include <string.h>
+
 #ifdef HAVE_CONFIG_H
 #include "config.h"
 #endif
--- cal3d-0.11.0/src/cal3d_converter.cpp~	2006-05-28 19:35:09.000000000 +0200
+++ cal3d-0.11.0/src/cal3d_converter.cpp	2008-11-25 10:58:13.000000000 +0100
@@ -9,6 +9,8 @@
 // your option) any later version.                                            //
 //****************************************************************************//
 
+#include <string.h>
+
 #include "cal3d/cal3d.h"
 
 #define SKELETON 0
